src/test/java/com/conveyal/gtfs/graphql/GTFSGraphQLTest.java

```diff
@@ -0,0 +1,229 @@
+package com.conveyal.gtfs.graphql;
+
+import com.conveyal.gtfs.TestUtils;
+import com.conveyal.gtfs.loader.FeedLoadResult;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import graphql.GraphQL;
+import org.apache.commons.io.IOUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.sql.DataSource;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.conveyal.gtfs.GTFS.createDataSource;
+import static com.conveyal.gtfs.GTFS.load;
+import static com.conveyal.gtfs.GTFS.validate;
+import static com.conveyal.gtfs.TestUtils.getResourceFileName;
+import static com.zenika.snapshotmatcher.SnapshotMatcher.matchesSnapshot;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class GTFSGraphQLTest {
```

Comment:
Please add Javadoc to all new classes explaining at least briefly what they do (in this case what sort of tests they contain).